R/valueButton.R

```diff
@@ -51,4 +48,30 @@ valueButton <- function(inputId, label, selector, attribute = "value", icon = NU
   )
 }
 
+#' @rdname valueButton
+#' @export
+valueLink <- function(inputId, label, selector, attribute = "value", icon = NULL, ...) {
+
+  shiny::tagList(
+    value_button_dependency(),
+    shiny::tags$a(
+      id = inputId,
+      class = "value-button",
+      href = "#",
+      `data-val` = NULL,
+      `data-value-attribute` = attribute,
+      `data-selector` = selector,
+      list(icon, label),
+      ...
+    )
+  )
+}
+
+value_button_dependency <- function() {
```

Comment:
Do we need to wrap this into function?

Reply:
Do you think it should be a static object?